testmod_crypto_gcrypt build error

```
CMake Error at src/plugins/crypto/CMakeLists.txt:192 (set_property):
  set_property given TEST names that do not exist:

    testmod_crypto_gcrypt
```

I think the whole line 192 is useless, why setting the property twice?

--- a/src/plugins/crypto/CMakeLists.txt
+++ b/src/plugins/crypto/CMakeLists.txt
@@ -189,7 +189,6 @@ if (ADDTESTING_PHASE)
 		add_plugintest (crypto_gcrypt
 				MEMLEAK
 				LINK_PLUGIN "<no>")
-		set_property (TEST "testmod_crypto_gcrypt" PROPERTY LABELS memleak)
 	endif ()
 
 	if (HAS_BOTAN_4SURE)

--- a/tests/valgrind.suppression
+++ b/tests/valgrind.suppression
@@ -1,5 +1,4 @@
-
-1;4402;0c{
+{
    <insert_a_suppression_name_here>
    Memcheck:Leak
    match-leak-kinds: reachable
@@ -127,11 +126,6 @@
    obj:*/libjvm.so
    obj:*/libjvm.so
 }
-
-
-
-
-
 {
    <insert_a_suppression_name_here>
    Memcheck:Addr4

